lib/workers/repository/model/commit-message-factory.ts

```diff
@@ -0,0 +1,51 @@
+import type { RenovateSharedConfig } from '../../../config/types';
+import type { CommitMessage } from './commit-message';
+import { CustomCommitMessage } from './custom-commit-message';
+import { SemanticCommitMessage } from './semantic-commit-message';
+
+type CommitMessageConfig = Pick<
+  RenovateSharedConfig,
+  | 'commitMessagePrefix'
+  | 'semanticCommits'
+  | 'semanticCommitScope'
+  | 'semanticCommitType'
+>;
+
+export class CommitMessageFactory {
+  private readonly config: CommitMessageConfig;
```

Comment:
any reason why this private field doesn't start with underscore but all others?

Reply:
Nope, fixed